docs/tempo/website/configuration/_index.md

```diff
@@ -532,7 +532,11 @@ storage:
 
             # optional.
             # enable if endpoint is http
-            [insecure: <bool>]          
+            [insecure: <bool>]
+
+            # optional.
+            # enable to skip verification of an HTTPs endpoint
+            [insecure_skip_verify: <bool>]
```

Comment:
I know that this definition looks similar to those near it within the file.  It could be improved by stating that a value of true causes the verification to be skipped,  and that the default is false. (Did I get the implications right?)

When I read this, I have to make a mental jump to associating "enable" with the boolean value "true." Easy for me, but perhaps more difficult for others.

Reply:
Learned something by reading your comment. :) I would not have caught the same thing.